### ISSUE TITLE
Any comment about upcoming Git Rev News edition 20

### ISSUE THREAD

**Issue by chriscool** [git/git.github.io#186]
A currently mostly empty draft is there:

https://github.com/git/git.github.io/blob/master/rev_news/drafts/edition-20.md

Feel free to comment in this issue, or to use the edit button (that looks like a pen) to edit and create a pull request with the changes you would like.

Thanks!

cc @tfnico @gitster

**Comment by jnareb** [comment]
Yet another link: [Deveo: List of equivalent commands in Git, Mercurial, and Subversion](http://blog.deveo.com/list-of-equivalent-commands-in-git-mercurial-and-svn/) - not sure if worth including.

More links to come.

**Comment by chriscool** [comment]
Thanks @jnareb!

**Comment by chriscool** [comment]
Let's publish this edition next Wednesday October 19th.

By the way sorry but I managed to make a mistake in the commit message of: https://github.com/git/git.github.io/commit/a9557011946b700dccb8b77fc9cfa8030571a831
It's October 19th not October 20th.

**Comment by jnareb** [comment]
* [Gittup](http://gittup.org/gittup/) is an entire(-ish) Linux distribution in git, where everything is built with [tup](http://gittup.org/tup/), a file-based build system.

**Comment by jnareb** [comment]
* [Lesser known Git commands](https://hackernoon.com/lesser-known-git-commands-151a1918a60#.50uadu14m) article by Tim Pettersen... well, most of those are well named aliases using less known options or combination of options (`push --force-with-lease`, `commit --amend --no-edit`, `stash --keep-index` / `--include-untracked` / `--all`, `status --short --branch`, `merge --no-ff`,...)

**Comment by jnareb** [comment]
- [GitHub Classroom as Textbook and Workbook](https://blog.skilstak.io/github-as-text-book-and-work-book-828ffada9542#.68a703qtb) by Rob Muhlestein on SkilStak blog

though this one might be too GitHub-specific.

**Comment by jnareb** [comment]
* [Git – Basic hygiene of working with repository – The #AIMway](http://www.allinmobile.co/git-basic-hygiene-of-working-with-repository/) (All In Mobile) by Jerzy Puchalski

I'm not sure if it was not present in previous Git Rev News...

**Comment by jnareb** [comment]
[Purposes, Concepts, Misfits, and a Redesign of Git](http://neverworkintheory.org/2016/09/30/rethinking-git.html) would need, I think, something more that just link. There was short discussion on Git mailing list. And there was result of this work: [Gitless](http://gitless.com).

**Comment by jnareb** [comment]
The "Git User's Survey 2016" is to end at 20 October 2016... one day after Git Rev News edition 20 - but I wouldn't have time to do the analysis anyway.

**Comment by jnareb** [comment]
- [ripgrep is faster than {grep, ag, **git grep**, ucg, pt, sift}](http://blog.burntsushi.net/ripgrep/) on Andrew Gallant's Blog; what's nice is (beside very comprehensive benchmark) explanation about how ripgrep is fast.

Maybe we could borrow some ideas for `git grep` / `git log --grep` / etc...?

**Comment by jnareb** [comment]
I don't know if [patchwork](http://jk.ozlabs.org/projects/patchwork/), a web-based patch tracking system for OSS projects, was presen in previous Git Rev News or not.

**Comment by jnareb** [comment]
- <https://github.com/book/git-test-repository> - Git repository full of special cases, for testing purposes

**Comment by chriscool** [comment]
> I don't know if patchwork, a web-based patch tracking system for OSS projects, was present in previous Git Rev News or not.

@jnareb No it hasn't appeared on Git Rev News yet. Thanks!

**Comment by tfnico** [comment]
I'm working on adding the links now. Thanks for all the tips, @jnareb! I had some of them bookmarked already, but there were some new ones there too. 

It would be cool if we had a "link pot" where everyone could easily add their own links, and easily check if it has been published or found earlier. I've been using diigo.com to collect the links personally, but unfortunately it starts to cost money when one wants to start more collaborative stuff.

**Comment by tfnico** [comment]
OK, links are in, @chriscool. Is it my turn to do the Mailchimp dance this time? :)

**Comment by chriscool** [comment]
@tfnico thanks for the links! And yeah it's your turn :-)

**Comment by jnareb** [comment]
I have submitted a few pull requests for 20th edition of Git Rev News: #193 , #194 and #195

**Comment by tfnico** [comment]
@jnareb I think the last two can be merged as well, but I'll wait for @chriscool since they're up in "his" part of the newsletter :)

**Comment by jnareb** [comment]
Wouldn't "Gerrit Summit 2016 is coming" link be better in (newly created) **Events** category?

**Comment by jnareb** [comment]
Yet another link:

- [Why kernel development still uses email [LWN.net]](http://lwn.net/Articles/702177/)

This is tangentially related, because:

- Linux kernel uses Git for development
- Git itself _also uses email_ for its development

**Comment by tfnico** [comment]
@jnareb added. Thanks! If you keep this up we'll have to make you curator (interested?) :)

**Comment by jnareb** [comment]
Yet another pul request #196 (take it or leave it)

**Comment by jnareb** [comment]
@tfnico interested... though I cannot guarantee that I will have time for Git Rev News each month.

**Comment by chriscool** [comment]
@jnareb: great! No problem if you don't have time each month. We will you add you to the team in the next edition and maybe @peff can already give you commit rights to the repo in GitHub.

**Comment by peff** [comment]
> and maybe @peff can already give you commit rights to the repo in GitHub.

Done.

**Comment by tfnico** [comment]
Nice! Welcome to the team, @jnareb!

**Comment by jnareb** [comment]
Yet another pull request: #197 

A question: how do you choose topics from git mailing list to be covered by Git Rev News?

**Comment by jnareb** [comment]
...and joined the team...

**Comment by tfnico** [comment]
MailChimp campaign has been sent! Note that there will be a problem some day with #198.

**Comment by chriscool** [comment]
Great, thanks!

**Comment by jnareb** [comment]
Should this issue be closed after Git Rev News 20?

**Comment by chriscool** [comment]
@jnareb people may find some problems in what we published so it might be better to not close it too early.

About the way I choose topics from the git mailing list, I think that the easiest and simplest for me is to write on a thread I found interesting when I read it. Sometimes I also choose topics that are involving many people and/or some specific people as I think it is nice to not always talk about the same contributors. I also like to talk about Git related talks at conferences especially the git merge.
Sometimes Junio or someone else points to a thread on one issue like the current one, and I think it could indeed be interesting for the current or the next edition.
